### PR TITLE
First commit 🌚

### PR DESCRIPTION
**Pull request by pschmitt**

### CODE REVIEW

File: Infoblox_Query/main.py [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+        .get("vlans", [])
+    )
+
+    old_vlan = check_repeat_number_vlan(previous_vlans, json_data)
+
+    if old_vlan is not None:
+        return int(old_vlan["vlan_id"]), ""
+
+    # Ok, we _have_ to create a whole new VLAN
+    if len(previous_vlans) > 1:
```
Comment (pschmitt) [inline Infoblox_Query/main.py:422]: Doesn't pylint / flake8 complain about this?

**Comment by JoaquimEsteves** [comment]
TODO:

* [ ] Finish Infoblox -> Device42 integration
    - It's the whole point of the script anyway
* [ ]  Refactor into smaller modules
    - [ ] Separate Typing stuff into they're own little `.pyi` files.
    - [ ] Separate Infoblox / Device42 functions into their own thing
    - [ ] Create a `__main__.py` that can be executed as a job every once in a while
* [ ] Remove config files and convert the functionality to [argparse](https://docs.python.org/3/library/argparse.html)
    - They make running scripts through *CI* tricky
    -  and Philipp doesn't like 'em

File: Infoblox_Query/.github/workflows/ci.yaml [hunk 2]
```diff
@@ -47,7 +47,7 @@ jobs:
 
       - name: isort
         run: |
-          isort --check --diff --verbose .
+          isort --check --diff --verbose
```
Comment (pschmitt) [inline Infoblox_Query/.github/workflows/ci.yaml]: This broke the check, man

**Comment by pschmitt** [comment]
Do we really need a `setup.py`?

File: Infoblox_Query/.github/workflows/ci.yaml (hunk 2 above)
Comment (JoaquimEsteves) [reply Infoblox_Query/.github/workflows/ci.yaml]: My fault! When I tried to run isort on my computer I was getting one of those lovely `DIR NOT FOUND` errors. I naively assumed I would get the same result here.

It seems to be a problem with my own version of isort. Restored it to it's previous functionality.

**Comment by JoaquimEsteves** [comment]
> Do we really need a `setup.py`?

it has been *purged*. The problem was just the py.typed that wasn't cooperating.

**Pull request merged**